corehq/apps/app_manager/suite_xml/sections/remote_requests.py

```diff
@@ -102,14 +102,13 @@ def _build_remote_request_queries(self):
         ]
 
     def _build_remote_request_datums(self):
-        details_helper = DetailsHelper(self.app)
         return [SessionDatum(
             id='case_id',
             nodeset=(CaseTypeXpath(self.module.case_type)
                      .case(instance_name=RESULTS_INSTANCE)),
             value='./@case_id',
-            detail_select=details_helper.get_detail_id_safe(self.module, 'case_short'),
-            detail_confirm=details_helper.get_detail_id_safe(self.module, 'case_long'),
+            detail_select=id_strings.detail(self.module, 'search_short'),
+            detail_confirm=id_strings.detail(self.module, 'case_long'),
```

Comment:
This looks suspicious - is it definitely okay that the long detail id in particular loses the checks in `get_detail_id_safe`?

Reply:
updated in the newest commit: https://github.com/dimagi/commcare-hq/pull/18533/commits/347fc2c67a199fb5ceb373dcf7abb427493019a9